### PR TITLE
Chore/dotenv

### LINKED ISSUE #2: Chore - Add dot env capabilities
## Checklist ✅

- [x] The issue is present in the latest release.
- [x] I have searched the issues of this repository and believe that this is not a duplicate.

## Summary 💡

With a .env file we should be able to load environments variables through a file, this will improve CI/CD of the project. 

## Requirements 🌈

Given some predefined environment variables are loaded into the .env file
then the system should load those variables. 

## Motivation 🔦

Improve the CI/CD Pipeline

### PR DESCRIPTION
**Pull request by dmorav1**
New pull request with suggested changes on branches

### CODE REVIEW

File: config/mongodb.go [hunk 1]
```diff
@@ -23,6 +24,10 @@ func ReadMongoDBConfig() *MongoDBConfig {
 }
 
 func getEnvVariableAsString(name string) string {
+	err := godotenv.Load()
+	if err != nil {
+		log.Fatal("Error loading .env file")
+	}
```
Comment (EnriqueePerez) [inline config/mongodb.go:30]: Just to be sure. Did you check that this really works as expected?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,94 @@
+
```
Comment (EnriqueePerez) [inline .gitignore:1]: Did you add the .env file to the .gitignore? It seems that it is not here. Please let me know if I didn't look for it well

**Review by EnriqueePerez — CHANGES_REQUESTED** [review]

File: config/mongodb.go (hunk 1 above)
Comment (dmorav1) [reply config/mongodb.go:30]: When .env is not present: 

image

File: .gitignore (hunk 2 above)
Comment (dmorav1) [reply .gitignore:1]: Now it is :)

**Review by EnriqueePerez — APPROVED** [review]

**Pull request merged**